### DISCUSSION TITLE
Problem mqtt, config.toml file

### DISCUSSION THREAD

**Discussion by psy501** [s-allius/tsun-gen3-proxy#197] [General]
Thanks for this integration, I tried to install it via docker but I still have an error with the config.toml file. In which directory is it? Do I have to edit it with nano?

My micro inverter IS TSUN TSOL-MX1000

**Comment by psy501** [comment]
It's ok the config.config.toml is in

config/config.toml

I have pv1 and pv2 yet, but power daily unknow.... 

![Screenshot_20241007-095433.png](https://github.com/user-attachments/assets/f189c98d-5f70-4e91-b31b-a9aa8a98c820)

**Comment by s-allius** [comment]
Hi psy501,

you are right. The config.toml file should be in the config dir. In this directory you will find a template config.example.toml. Rename this  into config.toml and then adapt it with your editor. Nano will be fine.

Cheers Stefan
